src/integration-test/java/com/commercetools/project/sync/util/IntegrationTestUtils.java

```diff
@@ -134,6 +136,10 @@ private static void deleteProjectData(@Nonnull final SphereClient client) {
     queryAndExecute(client, TypeQuery.of(), TypeDeleteCommand::of);
     queryAndExecute(client, InventoryEntryQuery.of(), InventoryEntryDeleteCommand::of);
     queryAndExecute(client, CartDiscountQuery.of(), CartDiscountDeleteCommand::of);
+    queryAndExecute(
+        client,
+        StateQuery.of().plusPredicates(QueryPredicate.of("builtIn=\"false\"")),
```

Comment:
small comment on why we exclude this `buildIn` will be nicer

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/6598b6cd134709f46740b1d3dc4a5ef956d00779